R/create_certificate.R

```diff
@@ -72,98 +129,189 @@ create_certificate <- function(cohort_name,
 #' \dontrun{
```

Comment:
In this example you should add `googlesheets4::gs4_auth()` before you call `read_sheet()`.